LiveLessons4/Lesson4_5_functionalProgramming.playground/contents.swift

```diff
@@ -20,19 +20,6 @@ roundedUpSavings
 let doubleDigits = transactions.filter { $0 >= 10.0 }
 doubleDigits
 
-// currying
```

Comment:
I don't think currying will be removed completely, rather, the syntax as it currently is to create a curried function will be removed. You would still be able to declare a function which can accept a parameter and return a function expecting the next parameter, as outlined in Detailed Design here (https://github.com/apple/swift-evolution/blob/master/proposals/0002-remove-currying.md). Please remove this commit from the PR and I will review again.